suite-native/module-accounts/src/screens/AccountSettingsScreen.tsx

```diff
@@ -15,10 +15,28 @@ import {
     StackProps,
 } from '@suite-native/navigation';
 import { BottomSheet, Box, Button, Card, Text, VStack } from '@suite-native/atoms';
-import { accountsActions, AccountsRootState, selectAccountByKey } from '@suite-common/wallet-core';
+import {
+    accountsActions,
+    AccountsRootState,
+    selectAccountByKey,
+    selectAccountLabel,
+} from '@suite-common/wallet-core';
 import { QRCode } from '@suite-native/qr-code';
 import { CryptoIcon } from '@trezor/icons';
-// import { deriveAddresses } from '@trezor/utxo-lib';
+
+import { AccountRenameModal } from '../components/AccountRenameModal';
+
+type AccoutSettingsScreenHeaderProps = {
+    accountName?: string;
+    accountKey: string;
+};
+
+const AccoutSettingsScreenHeader = ({
```

Comment:
This whole component seems redundant, it seems that you are just re-passing props. Use it directly.

Reply:
fixed in e44e7fee9